README.md

```diff
@@ -679,6 +721,8 @@ list(
 
 You can create as many `iks` or `openshift` clusters and worker pools on VPC. For `ROKS` clusters, make sure to enable public gateways to allow your cluster to correctly provision ingress application load balancers.
 
+NOTE: The cluster created will have a single worker pool with worker nodes distributed across 3 different subnet each in a different availability zone.
```

Comment:
- Will everyone understand why this is important? 
- Does this belong with HPCS?

Also, I'm not sure that it needs a `Note`. I think it could be just a paragraph.

Reply:
This is no longer the case. This statement can be removed.